features/prompt-history/tracking-groups.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: "Tracking Groups"
```

Comment:
Can we call this just "Groups". Please also update file names

Reply:
Just did.